google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineSupportProvider.java

```diff
@@ -187,7 +184,8 @@ private void addSupport(final Module module,
         LOG.error(ioe);
       }
       final Library library = addProjectLibrary(module, "AppEngine ORM",
-          Collections.singletonList(sdk.getOrmLibDirectoryPath()), sdk.getOrmLibSources());
+          Collections.singletonList(CloudSdkService.getInstance().getOrmLibDirectoryPath()),
```

Comment:
not a biggie, but unless I was worried that CloudSdkService could change mid thread, I'd be tempted to just extract a field or variable and reuse it across invocations.